makefiles/in_chroot/install.sh

```
@@ -55,39 +62,44 @@ function cdmake () {
 mkdir -p /usr/axiom/bin/
 for dir in $(ls -d software/sensor_tools/*/); do cdmake "$dir"; done
 for dir in $(ls -d software/processing_tools/*/); do cdmake "$dir"; done
+cdmake software/memtool
 
 mkdir -p /usr/axiom/script/
-for script in software/scripts/*.sh; do ln -sf $(pwd)/$script /usr/axiom/script/axiom-$(basename $script | sed "s/_/-/g"); chmod a+x $(pwd)/$script; done
-for script in software/scripts/*.py; do ln -sf $(pwd)/$script /usr/axiom/script/axiom-$(basename $script | sed "s/_/-/g"); chmod a+x $(pwd)/$script; done
+for script in $(cat <(ls software/scripts) software/scripts/bringup_scripts.list | sort | uniq -u | grep -E  '\.sh$|\.py$'); do ln -sf $(pwd)/software/scripts/$script /usr/axiom/script/$script; chmod a+x $(pwd)/software/scripts/$script; done
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)